src/app/handlers/blacklist.handler.ts

```diff
@@ -15,9 +22,15 @@ export class BlacklistHandler implements IHandler {
       return;
     }
 
-    this._expressions.forEach((expression) => {
-      if (message.content.toLowerCase().match(expression)) {
+    this._expressions.forEach(({regex, label}) => {
+      if (message.content.toLowerCase().match(regex)) {
         this.container.messageService.sendBotReport(message);
+        const rep = new Moderation.Report(
+          message.guild,
+          message.author.username,
+          `Please do not post ${label} links.`,
+        );
+        this.container.modService.fileReport(rep);
```

Comment:
I thought you also wanted to notify the user. This won't do that. It only files a report.